introduction.tex

```diff
@@ -22,14 +27,12 @@ \section{Methodology}
 
 
 \section{Time Scope and Future Projections}
-Because of the fast reactor transition scenario, the simulation should run to a distant
-future to demonstrate substantial change. In this case, the year is set from
-1970 to 2160. From 1970 to 2017, the existing data is used for the simulation.
-For the near future (\textasciitilde 2035), different projections and plans
-made by each governments are taken
-into account to estimate the direction of nuclear fleets in the EU (explained
-further in later sections). For the far future (2035 \textasciitilde 2100), an equilibrium capacity
-is set and decommissioned reactors are replaced by \gls{EPR}s or \gls{SFR}s.
+From the \gls{PRIS} data and other sources \cite{world_nuclear_2017} \cite{joskow_future_2012} \cite{hatch_politics_2013},
+the future projections of each EU nation's plan for their nuclear program has been assessed.
+The projections go up to 2050 at the latest. This justifies the simulation to take place from
+1970 to 2060, the latest foreseeable future. The specific plans for each EU nation is explained
```

Comment:
Maybe the confusion is that in your emails you have been thinking of MOX reactors and SFRs as distinct, but I haven't been. The description you gave of the paper you wanted to write was focused on mox -- to me, that included ASTRID SFRs. Importantly, the candidate SFR for France (the ASTRID) reactor, is a MOX fuelled SFR. Yes, SFRs are typically designed to have metal fuel, but the French ASTRID design hopes to use MOX specifically because of their long term history with mox fuel..